### PR TITLE
Axelg bitlocker

### PR DESCRIPTION
**Pull request by axelgMS**
Product Backlog Item 10310994: axelg to add "Bitlockerextension.log" to the Azure Disk Inspect Service

adding the Bitlocker extension log for Azure Security team

/WindowsAzure/Logs/Plugins/Microsoft.Azure.Security.AzureDiskEncryption/*/BitlockerExtension.log

### CODE REVIEW

File: docs/manifest_content.md [hunk 1]
```diff
@@ -742,6 +742,7 @@ diagnostic | copy | /WindowsAzure/GuestAgent\*/CommonAgentConfig.config
 diagnostic | copy | /WindowsAzure/Logs/Plugins/Microsoft.Compute.CustomScriptExtension/\*/\*.log
 diagnostic | copy | /WindowsAzure/Logs/Plugins/Microsoft.CPlat.Core.RunCommandWindows/\*/\*.log
 diagnostic | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.ActiveDirectory.AADLoginForWindows/\*/\*.l<br>og
+diagnostic | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.Security.AzureDiskEncryption/\*/BitlockerE<br>xtension.log
```
Comment (ejarvi) [inline docs/manifest_content.md:745]: `Is the <br> intentional in "BitlockerE<br>xtension.log" ? `

Comment (ejarvi) [reply docs/manifest_content.md:745]: Nevermind, I see this is markdown.

**Review by ejarvi — APPROVED** [review]
LGTM

**Comment by axelgMS** [comment]
#sign-off

**Review by chintanrp — APPROVED** [review]
LGTM

**Pull request merged**